internal/usage/consts.go

```diff
@@ -0,0 +1,18 @@
+package usage
+
+const (
+	//usage of additional ntp source feature
+	AdditionalNtpSourceUsage string = "Additional NTP Source"
```

Comment:
i don't see how any of the specific features are related to this package.

Reply:
I am avoiding using hard coded strings for feature names and it summarize the supported metrics